tests/test_get_all_sentences.rs

```diff
@@ -26,18 +26,14 @@ fn test_get_all_sentences_returns_200() {
     let second_english_iso639_3 = "fra";
```

Comment:
second_english_  ...  with value "fra" :) 

i think here the "_english_" is actuallly not useful for the test AND is erroneous